RepoDb.Core/RepoDb/IPrimaryOptions.cs

```diff
@@ -0,0 +1,10 @@
+using System.Data;
+
+namespace RepoDb
```

Comment:
There is a folder named [Interfaces](https://github.com/mikependon/RepoDb/tree/master/RepoDb.Core/RepoDb/Interfaces). Can you place all the interfaces there? The namespace must be `RepoDb.Interfaces`. Thanks

Reply:
Done, I will Push soon